extension/snippets.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Basic": {
+    "prefix": ["basic"],
```

Comment:
For these, I think we should have a super short version as well. The [Jest Snippets](https://marketplace.visualstudio.com/items?itemName=andys8.jest-snippets) extension that I use has 2/3 letter acronyms for each snippet.

![image](https://user-images.githubusercontent.com/37993418/183537913-e40ecaa3-6c37-4e0f-8909-7db945317ff6.png)

The longer prefix should also mention the fact that it is a snippet for a stage.

Here are some more examples:

- [ES7+ React/Redux/React-Native snippets](https://github.com/dsznajder/vscode-react-javascript-snippets/blob/HEAD/docs/Snippets.md) (5.6m)
- [VS Code JavaScript (ES6) snippets](https://github.com/xabikos/vscode-javascript)